Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionGenericCEF.yaml

```diff
@@ -0,0 +1,302 @@
+Parser:
+    Title:  Network Session ASIM parser for Generic CEF
+    Version: '0.1'
+    LastUpdated: Feb 17, 2022
+Product:
+    Name: Common Event Format
+Normalization:
+    Schema: NetworkSession
+    Version: '0.2.2'
+References:
+    - Title: ASIM Network Session Schema
+      Link: https://aka.ms/AzSentinelNetworkSessionDoc
+    - Title: ASIM
+      Link: https://aka.ms/AzSentinelNormalization
+    - Title: Micro Focus Security ArcSight Common Event Format [pdf]
+      Link: https://www.microfocus.com/documentation/arcsight/arcsight-smartconnectors/pdfdoc/common-event-format-v25/common-event-format-v25.pdf
+    - Title: CEF and CommonSecurityLog field mapping
+      Link: https://docs.microsoft.com/en-us/azure/sentinel/cef-name-mapping
+Description: |
+    This ASIM parser supports normalizing Common Event Format to the ASIM Network Session schema.
+ParserName: ASimNetworkSessionGenericCEF
+EquivalentBuiltInParser: _ASim_NetworkSession_GenericCEF
+ParserParams:
+    - Name: disabled
+      Type: bool
+      Default: false
+ParserQuery: |
+    let ASimNwCSLParsers = datatable(DeviceVendor:string, DeviceProduct:string)[
+      "Palo Alto Networks", "PAN-OS"
+      , "Zscaler" , "NSSFWlog"];
+    let ALLOW_TERMS = dynamic(['allow', 'allowed', 'permit', 'accept', 'start']);
+    let RESET_TERMS = dynamic(['reset', 'rst']);
+    let DeviceActions=datatable(DvcAction:string, OriginalDeviceAction:string )[
+       'Allow','allow' 
+      ,'Allow','allowed'
+      ,'Allow','permit'
+      ,'Allow','accept'
+      ,'Allow','start'
```

Comment:
Where did you find this one? 'start' usually implies a session start and not a device action. We use the EventSubType field for it.

Reply:
I understand that, but there are such events (try DeviceAction =~'start').
Maybe our (source and dest ips are not empty) filter is not tight enough.

Reply:
Try were?

Reply:
CyberSecuritySOC, DeviceAction=='START'.
fortgigate also has a start action

Reply:
Looked into that: 

'start' as a DvcAction does not make sense. DvcAction in ASIM implies the action taken on the connection. When a source sends start, it should go to the subtype. Why do they set it that way? I would put it on a mapping error we need to fix.

~ Ofer